Animated images: slide-in from left with staggered delays

LivayaaSharma opened the pull request:
Hey! 
For this issue, I implemented animation so the two images in the section slide in from the left with staggered delays (0.3s then 0.4s), following the same style used in the “Help in Courses” section which I used for reference. Let me know if you’d like me to adjust the delay timing or make any other tweaks, I’ll be happy to!
Ps. below I've attached what this looks like on md as well as lg screen orientation.

https://github.com/user-attachments/assets/403821ac-8c98-4c37-985d-6bda003c23c3


https://github.com/user-attachments/assets/ef9b383c-be2a-409c-a32a-604fc52a2b87

Kevinloritsch commented on src/components/home/Images.tsx:
```diff
@@ -5,21 +5,25 @@ import ulaPic2 from "@/public/home/ulaPic2.webp";
-        <Image src={ulaPic1} alt="Ula Picture 1" />
-        <Image src={ulaPic2} alt="Ula Picture 2" />
-      </motion.div>
+      <div className="flex w-5/6 flex-col justify-center gap-8 md:w-1/3 md:flex-row">
```
I think the image sizes got messed up on the larger screen, below is what they looked like before:

<img width="2582" height="1120" alt="image" src="https://github.com/user-attachments/assets/4283c8ef-feef-43ef-a779-e352d5ba6a54" />

LivayaaSharma commented:
I adjusted the layout so the images scale better on larger screens. I replaced the old md:w-1/3 (which was making them shrink) with w-5/6 max-w-5xl for the container, and set each image to md:w-1/2 so they split evenly on desktop while stacking on mobile. On my end, the images looked a bit off even before editing the first time, and I think it was coming from the md:w-1/3. Right now it looks close to the reference screenshot you shared but let me know if you’d like me to tweak the sizing further, like if you want them bigger still. :D 

https://github.com/user-attachments/assets/0f770a54-d8ca-4a50-b74e-96c59c2ce97f

Kevinloritsch reviewed (approved):
lgtm
